src/translate.rs

```diff
@@ -1003,6 +1003,88 @@ where
     }
 }
 
+pub struct PtrArray(*mut glib_sys::GPtrArray);
+
+impl Drop for PtrArray {
+    fn drop(&mut self) {
+        unsafe {
+            glib_sys::g_ptr_array_free(self.0, true.to_glib());
```

Comment:
`g_ptr_array_unref()` still

Reply:
I misunderstood the previous comment, sorry. Not it is using `g_ptr_array_unref()` instead of `g_ptr_array_free()`.

Reply:
Sorry for being not more clear!